src/utils/utils.mjs

```diff
@@ -0,0 +1,54 @@
+export function filterSeancesByFeatures(seances, emptyAmount, hasEmptyVip, hasEmptyDouble) {
+  let filterList = [];
+  emptyAmount && filterList.push(emptySeatsFilter);
+  hasEmptyVip && filterList.push(emptyVipFilter);
+  hasEmptyDouble && filterList.push(emptyDoubleFilter);
+
+  return filterList.reduce((acc, filter) => {
+    acc = filter(acc, emptyAmount);
+    return acc;
+  }, seances);
+}
+
+function emptyVipFilter(seances, emptyAmount) {
```

Comment:
emptyVipFilter and emptyDoubleFilter are basically the same
DRY

Reply:
fixed